src/wp-includes/class-http.php

```diff
@@ -459,7 +459,13 @@ public static function normalize_cookies( $cookies ) {
 
 		foreach ( $cookies as $name => $value ) {
 			if ( $value instanceof WP_Http_Cookie ) {
-				$cookie_jar[ $value->name ] = new Requests_Cookie( $value->name, $value->value, $value->get_attributes(), array( 'host-only' => $value->host_only ) );
+				$attributes                 = array_filter(
+					$value->get_attributes(),
+					function( $attr ) {
```

Comment:
Use `static` for performance

```suggestion
					static function( $attr ) {
```

Reply:
Fixed. This PR is now ready for commit.